ibm_catalog.json

```diff
@@ -0,0 +1,129 @@
+                    "description": "Yes"
+                }
+            ],
+            "flavors": [
+                {
+                    "label": "Regional bucket replication",
+                    "name": "replication",
+                    "install_type": "fullstack",
+                    "working_directory": "solutions/tf-fullstack-da",
+                    "compliance": {
+                    },
+                    "iam_permissions": [
+                        {
+                            "role_crns": [
+                              "crn:v1:bluemix:public:iam::::serviceRole:Manager"
+                            ],
+                            "service_name": "cloud-object-storage"
+                        }
+                    ],
+                    "architecture": {
+                        "descriptions": "This architecture supports creating and configuring 2 Object Storage buckets with replication.",
+                        "features": [
+                            {
+                                "title": "Optionally creates a resource group",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Creates an Object Storage instance",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Creates a source Object Storage bucket",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Creates a target Object Storage bucket",
+                                "description": "Yes"
+                            },
+                            {
+                                "title": "Configures a replication rule between buckets",
+                                "description": "Yes"
+                            }
+                        ],
+                        "diagrams": [
+                            {
+                                "diagram": {
+                                    "caption": "Object Storage buckets with replication",
+                                    "url": "https://raw.githubusercontent.com/terraform-ibm-modules/sample-deployable-architectures/main/reference-architecture/cos-replication.svg",
+                                    "type": "image/svg+xml"
+                                },
+                                "description": "This architecture supports creating and configuring Object Storage bucket replication."
+                            }
+                        ]
+                    }
+                },
+                {
+                    "label": "Static website configurator",
+                    "name": "static-website-configurator",
+                    "install_type": "extension",
```

Comment:
Same comment as above, would it make sense to add comments on this line and 92 to show the real difference here is that this variation requires a prereq to be deployed in the account already to move forward with the extension type?

Reply:
unfortunately not - json does not support comments. Anything we want to highlight will have to be done in Cloud docs